bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -960,9 +813,15 @@ spec:
           if [[ "{{.OVN_MULTI_NETWORK_POLICY_ENABLE}}" == "true" ]]; then
             multi_network_policy_enabled_flag="--enable-multi-networkpolicy"
           fi
+          
+          admin_network_policy_enabled_flag=
+          if [[ "{{.OVN_ADMIN_NETWORK_POLICY_ENABLE}}" == "true" ]]; then
+            admin_network_policy_enabled_flag="--enable-admin-network-policy"
+          fi
 
-          exec /usr/bin/ovnkube --init-node "${K8S_NODE}" \
+          exec /usr/bin/ovnkube --init-ovnkube-controller-with-node "${K8S_NODE}" \
```

Comment:
Didn't we have separate flags for this? Or are you using an old image?

Reply:
updating 2 files is too difficult :D thanks a lot for catching these, I will update the PR when the first CI round ends just to see the results.